commands/minecraft.js

```diff
@@ -0,0 +1,44 @@
+const Augur = require("augurbot"),
+  request = require("request-promise-native"),
+  minecraft = require("../utils/minecraftAPI");
+
+const Module = new Augur.Module()
+  .addCommand({
+    name: "minecraftskin",
+    description: "Gets the Minecraft skin of a user.",
+    syntax: "[@user]",
+    category: "Minecraft",
+    process: async (msg, suffix) => {
+      let user = false,
+        name = false;
+
+      if (u.userMentions(msg).size > 0) {
+        user = u.userMentions(msg).first();
+      } else if (!suffix) {
+        user = msg.author;
+      }
+
+      if (user) {
+        let ign = await Module.db.ign.find(user.id, 'twitch');
+        if (ign) name = encodeURIComponent(ign.ign);
+        else {
+          msg.channel.send(user + " has not set a Minecraft name with `!addign minecraft`.").then(u.clean);
```

Comment:
I think D.js 12 updated this to break. Probably should fix it elsewhere as well. 
```suggestion
          msg.channel.send(`${user} has not set a Minecraft name with \`!addign minecraft\`.`).then(u.clean);
```